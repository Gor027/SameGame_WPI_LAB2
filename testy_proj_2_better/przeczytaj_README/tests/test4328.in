 q qq __q  ___q_qqqqqq_ qqqq____  __ q  _ _qq qq
_q ___q  _  __ __qq qq_ _qq ___  q_q__qq  q_ _ _  _  _ __ _qq q  _qq q__ _  q
_q_  _q   _ q  q _q  q q   __ q   ___q_qq q _  
_q q_ qq_ _       _qq_ qqqq q_ q _qqq_q_q_qqq_ __q __   _
_ qq q_q
