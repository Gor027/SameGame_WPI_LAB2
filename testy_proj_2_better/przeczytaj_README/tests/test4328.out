     _
q q  q           q  q      _    __ q    _
q _q_q_    _  _ qq qq  qqq ___  _q__qq_ qqq  q
q_q__qqq _ __ _qqqqqq_q_qqq____q___q_qq q___ _       _
qqq_q_q____q_qq___qq__qqqq_q_qq_qqq_q_q_qqq_q__q_______qqq_qqq___q
